EssentialFeed/EssentialFeediOS/Feed UI/Controllers/CommentCellController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CommentCellController.swift
+//  EssentialFeediOS
+//
+//  Created by Anton Ilinykh on 09.03.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public final class CommentCellController {
+	private let model: Comment
+	
+	public init(model: Comment) {
+		self.model = model
+	}
+	
+	func view(for tableView: UITableView) -> UITableViewCell {
+		let cell = tableView.dequeueReusableCell(withIdentifier: "CommentCell") as! CommentCell
```

Comment:
You can use the provided helper to dequeue cells instead:

```suggestion
		let cell: CommentCell = tableView.dequeueReusableCell()
```